.circleci/config.yml

```diff
@@ -59,6 +62,38 @@ jobs:
                 name: Publish @pxblue/react-native-template-authentication-typescript
                 command: |
                   cd authentication-typescript && yarn publish:package -b $CIRCLE_BRANCH
+
+    tag:
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - gh/setup
+            - run:
+                name: Tag @pxblue/react-native-template-blank
+                command: |
+                  cd blank && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-blank
+            - run:
+                name: Tag @pxblue/react-native-template-blank-typescript
+                command: |
+                  cd blank-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-blank-typescript
+            - run:
+                name: Tag @pxblue/react-native-template-routing
+                command: |
+                  cd routing && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing
+            - run:
+                name: Tag @pxblue/react-native-template-routing-typescript
+                command: |
+                  cd routing-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing-typescript
+            - run:
+                name: Tag @pxblue/react-native-template-authentication
+                command: |
+                  cd authentication && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication
+            - run:
+                name: Tag @pxblue/react-native-template-authentication-typescript
+                command: |
+                  cd authentication-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication-typescript
```

Comment:
these suffixes should be the same as the package name (the cra- prefix is from the react packages)

Reply:
I've just removed the suffix; should be auto-populated now.

Reply:
I've just removed the suffix; should be auto-populated now.